extensions/nba-game-viewer/src/views/standings.tsx

```diff
@@ -29,7 +29,7 @@ const Standings = () => {
         >
           <List.Dropdown.Item value={Conference.Eastern} title="Eastern" />
           <List.Dropdown.Item value={Conference.Western} title="Western" />
-          <List.Dropdown.Item value={League} title="League" />
+          <List.Dropdown.Item value={League} title="Whole League" />
```

Comment:
Makes it less consistent as there's no sub leagues

Reply:
Happy to revert it.